feat(trace-view): Update our mocks to create traces

wmak opened the pull request:
- This adds the ability to set spans & parent_span_id when creating sample transactions, which will help with tests on traces as well
- Currently mocked transactions are only connected by their trace id instead of properly through spans. Which means that new trace view functionality would appear not to work

wmak committed:
feat(trace-view): Update our mocks to create traces

- This adds the ability to set spans when creating sample transactions
- Currently mocked transactions are only connected by their trace id
  instead of properly through spans. Which means that new trace view
  functionality would appear not to work

markstory commented on bin/load-mocks:
```diff
@@ -748,6 +704,9 @@ def main(num_events=1, extra_events=False, load_transactions=False):
 
         OrganizationAccessRequest.objects.create_or_update(member=dummy_member, team=team)
 
+    if load_transactions:
```
Any reason to not make this default to on? Having transactions by default simplifies onboarding and daily usage.

Zylphrex replied:
From personal experience, loading transactions is slow and prone to crashing clickhouse even with the `slow` flag. So I would hold off on making this the default until that's addressed.

k-fish replied:
Wait really? 336 transactions will crash clickhouse locally? Maybe we can load a small handful by default to make at least something appear for onboarding like Mark mentioned, and only load the rest of them to create a trend as their own option.

markstory commented on bin/load-mocks:
```diff
@@ -759,6 +718,131 @@ def main(num_events=1, extra_events=False, load_transactions=False):
+                    for name, backend_span_id in backend_span_ids
+                ],
+            )
+            if slow:
```
When would you run this without slow?

wmak replied:
I... will add some docs explaining this, but basically depends on your personal computer. I found that I could run `--load-transactions` consistently without crashing, but @Zylphrex has reported that running it crashes clickhouse consistently 😞

k-fish reviewed (approved):
Looks good.

Not blocking, but as a mix of what Mark and Tony were talking about, I think we should probably include a small handful of transactions as a default, and offer the trend transactions or more comprehensive transaction mocks as their own option.

wmak committed:
ref: Load 1 trace by default

- Load_trends replaces load_transactions
- Updated slow comment

wmak committed:
Merge branch 'master' into feat/mock-traces

wmak committed:
ref: Defaulting parent_span_id in a more reasonable way

- Only do the auto set for create_sample_event since most tests won't
  want to set the parent span themself
